core/src/main/java/tc/oc/pgm/filters/dynamic/FilterMatchModule.java

```diff
@@ -105,6 +121,9 @@ public void onMatchLoad(MatchLoadEvent event) {
               }
               this.registerListenersFor(filter.getRelevantEvents());
             });
+
+    // These two have to be on separate passes, in case a reactor wants to register a child filter
+    // for dynamic listening
```

Comment:
these what? there's nothing following this comment

Reply:
Its referring to the code above the comment.

Reply:
The normal thing is to have the comment directly above the code it is explaining, not after it, so move it to where it belongs